### ISSUE TITLE
dehydrated space carp plushies can breed

### ISSUE THREAD

**Issue by factoryman942** [tgstation/tgstation#39371]
the child plushies are also dehydrated space carp, meaning you can double your fish in an unintended way

**Comment by AnturK** [comment]
This is pretty funny but probably shouldn't happen.

**Comment by Kagus** [comment]
I mean, the dehydrated carp _does_ have `//Child of carpplushie because this should do everything the toy does and more` commented on it... So this was either intended, or the person responsible just didn't know how much the toy could actually do.

As a side note, I... Suppose this also means you can tear the stuffing out of them and shove a grenade in?

**Comment by RandomMarine** [comment]
>I mean, the dehydrated carp does have //Child of carpplushie because this should do everything the toy does and more commented on it... So this was either intended, or the person responsible just didn't know how much the toy could actually do.

That comment existed before the breeding feature was added.

**Comment by Kerbin-Fiber** [comment]
>Pet plushie
>Remove stuffing
>Stuff Dehydrated Carp Plushie with a bee bomb
>Water plushie
>Carp attacks greyshit
>Greyshit kills carp
>Carp explodes
>Bees everywhere

Why isn't this a feature

**Comment by CosmicScientist** [comment]
>That comment existed before the breeding feature was added.

Correct. Unfortunately both had the same creator who wishes very much that he had foreseen this and taken advantage of it at every opportunity.
